### PR TITLE
Release Process + Term Refinement

### PR DESCRIPTION
**Pull request by CooperStansbury**
1. Updated terms (to a hopefully stable point) based on several discussions with @zachll and @grosscol.
1. Ran `tools/cpo_release.sh` to control `dictionary.md` + manage ontology version annotation
1. Fixed bonkers annotation properties.

### CODE REVIEW

**Review by grosscol — COMMENTED** [review]
I have no comments for the added/removed terms.

The causal_pathways.json is a mock up of the general structure of a json-ld representation of a causal pathway?

What is the purpose of the release_log and release_txt?  These look like intermediate artifacts from the release process.  Why commit them to the repo?

**Comment by CooperStansbury** [comment]
1. Yes - we'll discuss the json.ld file tomorrow. I just needed a place to put it. 
1. My thought was a file is slightly more accessible than commit history. I'm open to throwing report messaging to stdout instead of logging file.

**Comment by grosscol** [comment]
Looks good.  Squash?

File: tools/cpo_release.sh [hunk 1]
```diff
@@ -39,6 +39,9 @@ robot report --input ${CPO}
 # update dictionary as part of release process
 ${SCRIPT_DIR}/make-dict.sh
 
+# spell-check dictionary post release
+aspell check ${SCRIPT_DIR}/DICTIONARY.md
```
Comment (grosscol) [inline tools/cpo_release.sh]: You're going to want to add a check that aspell is available in a similar way that robot is checked for.

Comment (CooperStansbury) [reply tools/cpo_release.sh]: I know, I'm working on this now...

**Comment by CooperStansbury** [comment]
@grosscol made some changes to the release process, including compatibility checking for `aspell`.

`cpo_release.sh` would `rm` the file if there were failures, and this isn't good. 

please review and I will happily squash once you sign off. 

Thanks!

File: tools/cpo_release.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# spell checker
+< ${SCRIPT_DIR}/../DICTIONARY.md aspell list | grep -f /dev/stdin -C 1 ${SCRIPT_DIR}/../DICTIONARY.md
+
+echo ""
+
+# clean up
+while true; do
+    read -p "Are you ready to release this version?" yn
+    case $yn in
+        [Yy]* ) rm cpo.owl; mv cpo-updated.owl cpo.owl; rm catalog-v001.xml; echo 'RELEASE COMPLETE.'; break;;
```
Comment (grosscol) [inline tools/cpo_release.sh:62]: Why remove the catalog file?  Xml files currently ignored by git (.gitignore) so it won't end up in the repo anyway.

**Review by grosscol — COMMENTED** [review]
This looks fine.  Only one question about removing the  catalog.

The user prompt at the end is overkill, but not a bad kind of overkill. Git reset provides the same effect.

File: tools/cpo_release.sh (hunk 2 above)
Comment (CooperStansbury) [reply tools/cpo_release.sh:62]: Just for the local system. I find myself removing this file a lot.

Comment (grosscol) [reply tools/cpo_release.sh:62]: Interesting.  I would not expect that the catalog needs to be removed and recreated frequently.  If removing it becomes a problem for other users, we can fix it then.

**Comment by CooperStansbury** [comment]
@grosscol squashed. please merge!

**Pull request merged**